model/patch/patch.go

```diff
@@ -152,6 +160,7 @@ type Patch struct {
 	BackportOf      BackportInfo           `bson:"backport_of,omitempty"`
 	MergePatch      string                 `bson:"merge_patch"`
 	GithubPatchData thirdparty.GithubPatch `bson:"github_patch_data,omitempty"`
+	GitInfo         *GitMetadata           `bson:"git_info"`
```

Comment:
This can also be omitempty

Reply:
If we only store this information for commit queue patches, is it worth calling it CommitQueueGitInfo or something like that? I'm a bit concerned that it will be confusing for developers to not have this populated for other patches..

Reply:
Made this `omitempty` too.
My intention is that GitInfo is actually not stored for commit queue patches 🙃 , but rather with all "regular" CLI patches so we can make commit queue patches out of them.